Makefile

```diff
@@ -1,5 +1,5 @@
 shell: ## run a shell on the learner-portal container
-	docker exec -it /bin/bash
+	docker exec -it frontend-app-learner-portal_web_1 /bin/bash
```

Comment:
I mentioned it in https://github.com/edx/frontend-app-learner-portal/pull/7 but I suggest we add a container_name to the docker-compose.yml (see devstack for examples) so that we can give this a more consistent name.